src/TConsole.cpp

```diff
@@ -2483,12 +2483,16 @@ bool TConsole::raiseWindow(const QString& name)
 {
     auto pC = mSubConsoleMap.value(name);
     auto pL = mLabelMap.value(name);
+    auto pM = mpMapper;
     if (pC) {
         pC->raise();
         return true;
     } else if (pL) {
         pL->raise();
         return true;
+    } else if (pM && name.toLower() == QLatin1String("mapper")) {
```

Comment:
CLazy will pick up on this - redundant-use-of-toUpper/toLower - the QString::compare() method has case-insensitive form that takes a `QLatin1String` - so you can change it to:
```CPP
     } else if (pM && !name.compare(QLatin1String("mapper"), Qt::CaseInsensitive)) { 
```

Same thing also applies once more further down.

Reply:
:thinking: However - how do we **know** that the map widget will be called `mapper`?

Reply:
Thanks for all the suggestions, I commited them.

The choose the name "mapper" for the map widget because there is only one unique map widget possible per profile so there should be no way to get confused.

Problems could arise if someone has a label, subconsole with the name "mapper". But then there is the same discussion for it if someone names its subconsole/label "main".

And before Mudlet 4.6.1 it was even possible to have the same name for labels and consoles. (I'm not sure if it's still possible somehow)

An alternative to the name "mapper" would be to make new functions only for the mapper like
raiseMapper/lowerMapper.